setup.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+"""
+# *****************************************************************
+# (C) Copyright IBM Corp. 2021. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *****************************************************************
+"""
+from setuptools import find_packages, setup
+import open_ce
+
+def get_readme():
+    with open("README.md") as f:
+        return f.read()
+
+
+CONSOLE_SCRIPTS = [
+    "open-ce = open_ce.opence:main",
+]
+
+REQUIRED_PACKAGES = [
+    "pyyaml",
+]
+
+setup(
+    name="open-ce",
+    version=open_ce.__version__,
+    description="Open-CE tools for building feedstocks",
+    long_description=get_readme(),
+    url="https://github.com/open-ce/open-ce",
+    author="Open-CE Dev Team",
+    author_email="",
+    packages = find_packages(),
+    scripts = ["open_ce/open-ce"],
```

Comment:
Instead of scripts, we probably want to define this as the enrty point:
```
entry_points={
        'console_scripts': [
            'open-ce = open-ce:main',
        ],
    },
```

Something along those lines.

Reply:
I'd tried this earlier. I've also CONSOLE_SCRIPTS defined above in the setup.py (which I forgot to clean). But it didn't work. Earlier it was not putting the `open-ce` into `bin` directory and then it failed to start.

Reply:
Okay, what you have is working. If we find there is a better way to do it later we can always change it.